Asaas/grails-app/controllers/asaas/PayerController.groovy

```diff
@@ -1,44 +1,47 @@
 package asaas
 
+import asaas.adapter.PayerSaveAdapter
 import asaas.Customer
 import asaas.Payer
+import asaas.Payer
 import asaas.PayerService
 
 import grails.validation.ValidationException
 
 class PayerController {
-  
-  def payerService
-  
-  def index() {
-    List<Customer> customers = Customer.list()
-    [customers: customers]
-  }
-
-  def save() {
-    try {
-      Payer payer = payerService.save(params)
-      redirect(action:"show", id:payer.id)
-
-    } catch (ValidationException e) {
-      String errorsMessage = e.errors.allErrors.defaultMessage.join(", ")
-      flash.error = "Não foi possível salvar um pagador: $errorsMessage"
-      render(view: "show", params: params)
-    }
-  }
 
-  def show() {
-    Payer payer = Payer.read(params.id.toLong())
+    def payerService
+
+    def index() {
+        List<Customer> customers = Customer.list()
+        [customers: customers]
+    }
 
-    if (payer) {
-      return [payer: payer]
+    def save() {
+        try {
+            PayerSaveAdapter payerSaveAdapter = new PayerSaveAdapter(params)
+            Payer payer = payerService.save(payerSaveAdapter)
+            redirect(action:"show", id:payer.id)
+
```

Comment:
Linha em branco no final de bloco assim é desnecessária, se for entre regras para separar blocos de regras, tudo bem
```suggestion
```